core/src/main/java/io/aiven/klaw/service/TopicSyncControllerService.java

```diff
@@ -521,10 +521,7 @@ public ApiResponse updateSyncBackTopics(SyncBackTopics syncBackTopics) {
     logArray.add("Type of Sync " + syncBackTopics.getTypeOfSync());
 
     if (commonUtilsService.isNotAuthorizedUser(getPrincipal(), PermissionType.SYNC_BACK_TOPICS)) {
-      return ApiResponse.builder()
-          .success(false)
-          .message(ApiResultStatus.NOT_AUTHORIZED.value)
-          .build();
+      return ApiResponse.notOk(ApiResultStatus.NOT_AUTHORIZED.value);
```

Comment:
We might be able to also create a ApiResponse.NOT_AUTHORIZED as it is a frequently used enum reponse.